packages/ngx-deploy-npm/project.json

```diff
@@ -82,6 +82,20 @@
       },
       "dependsOn": ["build"]
     },
+    "yarn-link-it": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "cd dist/packages/ngx-deploy-npm && npx --yes yarn link"
+      },
+      "dependsOn": ["build"]
+    },
+    "yarn-unlink-it": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "cd dist/packages/ngx-deploy-npm && npx --yes yarn unlink"
+      },
+      "dependsOn": ["build"]
```

Comment:
Shouldn't be?

```suggestion
      "dependsOn": ["^build"]
```

Reply:
Same as https://github.com/bikecoders/ngx-deploy-npm/pull/527#discussion_r1312432784